### PR TITLE
Update pre_release_version_bump.yml

### PR DESCRIPTION
**Pull request by Jaybsoni**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [ ] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [ ] Ensure that the test suite passes, by running `make test`.

- [ ] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [ ] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**

**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

File: .github/workflows/pre_release_version_bump.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v3
         with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (antalszava) [inline .github/workflows/pre_release_version_bump.yml:32]: How about adding in the push trigger temporarily? Once it runs once successfully, we should be good to go. :slightly_smiling_face: 
```
  push:
    branches:
      - version_bump_error
```

Comment (Jaybsoni) [reply .github/workflows/pre_release_version_bump.yml:32]: Good point !

Comment (Jaybsoni) [reply .github/workflows/pre_release_version_bump.yml:36]:
```suggestion
          # token: ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/pre_release_version_bump.yml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 name: Manually Triggered Version Bumps
 
 on:
-  workflow_dispatch:
+  push:
+    branches:
+      - version_bump_error
+
+#   workflow_dispatch:
```
Comment (Jaybsoni) [inline .github/workflows/pre_release_version_bump.yml:8]:
```suggestion
  workflow_dispatch:
```

File: .github/workflows/pre_release_version_bump.yml [hunk 3]
```diff
@@ -29,9 +33,10 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v3
         with:
+          # token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Jaybsoni) [inline .github/workflows/pre_release_version_bump.yml:36]:
```suggestion
```

**Review by antalszava — APPROVED** [review]
Looks good :muscle: :slightly_smiling_face:

File: .github/workflows/post_release_version_bump.yml [hunk 4]
```diff
@@ -36,4 +36,4 @@ jobs:
           body: updated changelog and _version.py
           branch: post-release-version-bump
           reviewers: antalszava
-          base: main
+          base: master
```
Comment (mlxd) [inline .github/workflows/post_release_version_bump.yml:39]: I think this is something we can change soon when time permits.

**Review by mlxd — APPROVED** [review]
All good -- merge away!

**Pull request merged**